database/migrations/20220531100155-update-sites.js

```diff
@@ -0,0 +1,195 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    const transaction = await queryInterface.sequelize.transaction()
+    try {
+      await queryInterface.dropTable("site_members", { transaction })
+      await queryInterface.dropTable("sites", { transaction })
+
+      await queryInterface.createTable(
+        "sites",
+        {
+          id: {
+            allowNull: false,
+            autoIncrement: true,
+            primaryKey: true,
+            type: Sequelize.BIGINT,
+          },
+          repository_name: {
+            allowNull: false,
+            unique: true,
+            type: Sequelize.TEXT,
+          },
+          created_by: Sequelize.TEXT,
+          created_at: {
+            type: Sequelize.DATE,
+            allowNull: false,
+            defaultValue: Sequelize.fn("NOW"),
+          },
+          updated_at: {
+            type: Sequelize.DATE,
+            allowNull: false,
+            defaultValue: Sequelize.fn("NOW"),
+          },
+          agency: Sequelize.TEXT,
+          site_name: Sequelize.TEXT,
+          contact: Sequelize.TEXT,
+          repository_url: Sequelize.TEXT,
+          hosting_id: Sequelize.TEXT,
+          staging_url: Sequelize.TEXT,
+          production_url: Sequelize.TEXT,
+          live_domain: Sequelize.TEXT,
+          redirect_from: Sequelize.JSONB,
+          uptime_id: Sequelize.TEXT,
+          uptime_url: Sequelize.TEXT,
+          launched_at: Sequelize.DATE,
+          launched_by: Sequelize.TEXT,
```

Comment:
I thought it would be easier to maintain a system if there were fewer tables. As a system grows, I often find myself needing to grab bits of information that I didn't think I would need. It's harder to make such changes if information is be spread between multiple tables for historical reasons.

Reply:
i think this is q a flawed argument - see this [link](https://exceptionnotfound.net/god-objects-the-daily-software-anti-pattern/); for an extension of your argument, this implies that we could simply lump everything (sites, site users, users etc etc) into a single table and be done with it